packages/truss/src/sections/tachyons/flexbox.ts

```diff
@@ -115,6 +115,21 @@ export const flexbox: CreateMethodsFn = () => [
     "pc",
   ),
 
+  ...newMethodsForProp(
+    "placeItems",
```

Comment:
@TylerR909 I had this on my local machine from a PR of yours a few weeks ago that used `placeItems` iirc...maybe :shrug: Either way fyi. :-)

Reply:
inb4 the next "Why aren't you using this? I added it!" but we didn't version bump.

Reply:
Good call: https://github.com/homebound-team/beam/pull/893